platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r86/ProblemProgressEventCrossVersionTest.groovy

```diff
@@ -68,30 +63,12 @@ class ProblemProgressEventCrossVersionTest extends ToolingApiSpecification {
         return listener.problems.collect { it.descriptor }
     }
 
-    static def assertProblemDetailsForTAPIProblemEvent(List<?> problems, String expectedDetails = null, String expectedDocumentation = null) {
-        assertProblemDetailsForTAPIProblemEventWithoutSolution(problems, expectedDetails, expectedDocumentation)
-        problems[0].solutions[0].solution == 'try this instead'
+    static def assertProblemDetailsForTAPIProblemEvent(List<?> problems) {
+        assertProblemDetailsForTAPIProblemEventWithoutSolution(problems)
     }
 
-    static void assertProblemDetailsForTAPIProblemEventWithoutSolution(List<?> problems, String expectedDetails, String expectedDocumentation) {
+    static void assertProblemDetailsForTAPIProblemEventWithoutSolution(List<?> problems) {
```

Comment:
Does it bring any value to keep this method? If it's trivial, I'd just inline the `problems.size() == 1` assertion.